site/reference/pkg/tree/README.md

```diff
@@ -97,7 +97,7 @@ kubectl get all -o yaml | kpt pkg tree \
 ### Synopsis
 <!--mdtogo:Long-->
 ```
-kpt pkg tree [DIR | -<STDIN>] [flags]
+kpt pkg tree [DIR | -] [flags]
```

Comment:
Where is the Synopsis do we explain what `-` means? This applies to all commands not just this.

Reply:
I will do the reference docs clean-up in coming PRs.